src/MLOps.NET/Storage/EntityResolvers/ModelSchemaResolver.cs

```diff
@@ -0,0 +1,20 @@
+﻿using MLOps.NET.Entities.Impl;
+using MLOps.NET.Storage.Interfaces;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace MLOps.NET.Storage.EntityResolvers
+{
+    internal sealed class ModelSchemaResolver : IEntityResolver<ModelSchema>
```

Comment:
I don't think we need this specific resolver, but we may want to add it to the `RunResolver`